### ISSUE TITLE
Can't close details, X press not registering

### ISSUE THREAD

**Issue by milosm** [zanna-37/hass-swipe-navigation#76]
Hi, 

First, I want to say I really like the functionality provided by this repo. It's a productivity timesaver, so thank you for developing and maintaining it.

I do have an issue that I couldn't solve by myself. The issue is that sometimes, after opening an entity detail window on mobile, I can't close it by clicking on the x button in the top left corner, as nothing happens when the x is pressed, like it's not registering the press. This happens on mobile, I'm using an iPhone. When this happens, I have to kill and restart the app to use it again, as I haven't found anything else that would close the details.

I have found that if I first pull down from the top so that Home Assistant first does a refresh, this doesn't happen as often, or it then only happens after some time, I'm assuming after leaving the app and coming back to it later.

I have reset the frontend cache from the companion app settings. Made no difference.

When I remove this repo from hacs and my Home Assistant, this issue never happens.

Is anyone experiencing something similar?

Thank you for your help!

<img src="https://github.com/zanna-37/hass-swipe-navigation/assets/411722/f3195429-9376-42d2-8ade-5eb595b13b30" width="400">

**Comment by zanna-37** [comment]
This is very strange, what's you configuration, if any?
Did you set the `prevent_default` to `true`?

**Comment by milosm** [comment]
No configuration.

**Comment by zanna-37** [comment]
I've never experienced anything like that. I doubt that this repo is the culprit.
In order to address your situation I need to reproduce the issue. But if this happens rarely it's very difficult for me to understand what's going on.

**Comment by zanna-37** [comment]
Are you still experiencing the issue?

**Comment by milosm** [comment]
Yes, still from time to time, unfortunately. I haven't figured it out yet.

**Comment by VoidElle** [comment]
Adding a +1.
Also on iOS, I have tried with an android device but the problem seems to not occur there.
Without the repo the problem is gone, I also have no configuration.

**Comment by zanna-37** [comment]
Is this still an issue?

**Comment by milosm** [comment]
Happy to report it’s not an issue for me anymore. At one point, can't pinpoint with what changes or update, it started working as expected.

Thanks for following up!

**Comment by zanna-37** [comment]
Honestly, I don't think it's related to this plugin, as there hasn't been any update that could affect it. But I'm happy it's not an issue anymore.
